flycheck.el

```diff
@@ -9048,9 +9059,15 @@ See URL `https://racket-lang.org/'."
                (t '(bold error)))))))
   :error-filter
   (lambda (errors)
-    (flycheck-sanitize-errors (flycheck-increment-error-columns errors)))
+    (flycheck-sanitize-errors
+     (flycheck-increment-error-columns
+      (seq-remove (lambda (err)
+                    (string= "/usr/share/racket/pkgs/compiler-lib/compiler/commands/expand.rkt"
```

Comment:
Ah, I seem to have mungled the version test for running `make check`.  I'll correct that and make sure it is failing this time, then wrap the line.

Reply:
`make check` now runs for 25.3 as intended.  It does not complain about the long line, and apparently never did.  We can add it, but it can be in another PR :)